tests/bulkwritecommand/bulkwritecommand-debug-001.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\BulkWriteCommand debug output
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+$bulk = new MongoDB\Driver\BulkWriteCommand([
+    'bypassDocumentValidation' => true,
+    'comment' => 'foo',
+    'let' => ['foo' => 1],
+    'ordered' => true,
+    'verboseResults' => true,
+]);
+
+var_dump($bulk);
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+object(MongoDB\Driver\BulkWriteCommand)#%d (%d) {
+  ["bypassDocumentValidation"]=>
+  bool(true)
+  ["comment"]=>
+  string(3) "foo"
+  ["let"]=>
+  object(stdClass)#%d (%d) {
+    ["foo"]=>
+    int(1)
+  }
+  ["ordered"]=>
+  bool(true)
+  ["verboseResults"]=>
+  bool(true)
+  ["session"]=>
+  NULL
+}
```

Comment:
The `BulkWriteCommand` class doesn't expose the list of operations in `var_dump` or xdebug. The only way to see its state is by executing it and using a subscriber to get the raw command.
Can we add a (virtual) property to show them or is there a limitation in libmongoc?